tests/test_stationsBlockedDetailViews.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from django.contrib.auth.models import User
+from rest_framework.test import APIRequestFactory
+from rest_framework.utils import json
+
+from BikeRentalApi import models
+from BikeRentalApi.enums import StationState
+from BikeRentalApi.models import BikeStation
+from BikeRentalApi.views import stations_blocked_detail
+
+
+@pytest.mark.django_db
+class TestBikesDetailViews:
+
+    @pytest.fixture
+    def tech(self):
+        tech = User.objects.create(
+            username = 'Mariusz', first_name = 'Mariusz', last_name = 'Tester', email = 'mariusz@test.com',
+            password = 'test123')
+        return models.Tech.objects.create(user = tech)
+
+    @pytest.fixture
+    def user(self):
+        user = User.objects.create(
+            username = 'Janek', first_name = 'Janek', last_name = 'Tester', email = 'Janek@test.com',
+            password = 'test1234')
+        return models.AppUser.objects.create(user = user)
+
+    @pytest.fixture
+    def admin(self):
+        user = User.objects.create(
+            username = 'Pawel', first_name = 'Pawel', last_name = 'Tester', email = 'Pawel@test.com',
+            password = 'test1234')
+        return models.Admin.objects.create(user = user)
+
+    @pytest.fixture
+    def station_working(self):
+        return BikeStation.objects.create(name = 'Test working station', state = StationState.Working)
+
+    @pytest.fixture
+    def station_blocked(self):
+        return BikeStation.objects.create(name = 'Test blocked station', state = StationState.Blocked)
+
+    @pytest.fixture
+    def factory(self):
+        return APIRequestFactory()
+
+    def test_delete_station_blocked_detail_user_status(self, factory, station_blocked, user):
+        request = factory.delete(f'/api/stations/blocked/{station_blocked.pk}')
+        request.headers = {'Authorization': f'Bearer {user.user.username}'}
+
+        response = stations_blocked_detail(request, station_blocked.pk)
+        assert response.status_code == 403
```

Comment:
Please change the asserted status codes in tests to 'status' enum to maintain consistency with Maciek's changes

Reply:
done